addon-test-support/-private/filter-test-modules.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Return the matched test.
+ * e.g. if an input is '!/weight/' it returns an array, ['!/weight/', '!', 'weight', ''];
+ *
+ * @param {*} modulePathFilter
+ */
+function getRegexFilter(modulePathFilter) {
+  return /^(!?)\/([\w\W]*)\/(i?$)/.exec( modulePathFilter );
```

Comment:
rather then having a function for this, can we instead:

* `const MODULE_PATH_REGEXP = /.../` at the top of the file, so the regexp can be reused?
* use `MODULE_PATH_REGEXP.exec(modulePathFile)` inline? That way, one can clearly see what it does (hiding this small line behind a function appears as unneeded indirection.)